### ISSUE TITLE
Latest 15 Papers - April 25, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#7]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Geoscience
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[GeoRSMLLM: A Multimodal Large Language Model for Vision-Language Tasks in Geoscience and Remote Sensing](http://arxiv.org/abs/2503.12490v1)** | 2025-03-16 | [papers-cool](https://papers.cool/arxiv/2503.12490v1) |  |
| **[Towards Foundation Models: Evaluation of Geoscience Artificial Intelligence with Uncertainty](http://arxiv.org/abs/2501.14809v1)** | 2025-01-15 | [papers-cool](https://papers.cool/arxiv/2501.14809v1) |  |
| **[When Geoscience Meets Foundation Models: Towards General Geoscience Artificial Intelligence System](http://arxiv.org/abs/2309.06799v5)** | 2024-11-12 | [papers-cool](https://papers.cool/arxiv/2309.06799v5) | <details><summary>accpe...</summary><p>accpeted by IEEE Geoscience and Remote Sensing Magazine</p></details> |
| **[Prototype-Based Methods in Explainable AI and Emerging Opportunities in the Geosciences](http://arxiv.org/abs/2410.19856v1)** | 2024-10-22 | [papers-cool](https://papers.cool/arxiv/2410.19856v1) | <details><summary>Accep...</summary><p>Accepted at AI for Science Workshop-Oral (Attention Track), Proceedings of 41st International Conference on Machine Learning (ICML) 2024</p></details> |
| **[Interpretable Geoscience Artificial Intelligence (XGeoS-AI): Application to Demystify Image Recognition](http://arxiv.org/abs/2311.04940v2)** | 2024-05-07 | [papers-cool](https://papers.cool/arxiv/2311.04940v2) | <details><summary>there...</summary><p>there are some erros in the results, and a newer revision is still preparing</p></details> |
| **[GeoGalactica: A Scientific Large Language Model in Geoscience](http://arxiv.org/abs/2401.00434v2)** | 2024-04-13 | [papers-cool](https://papers.cool/arxiv/2401.00434v2) |  |
| **[A novel approach for estimating functions in the multivariate setting based on an adaptive knot selection for B-splines with an application to a chemical system used in geoscience](http://arxiv.org/abs/2306.00686v3)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2306.00686v3) | 29 pages, 27 figures |
| **[When Geoscience Meets Generative AI and Large Language Models: Foundations, Trends, and Future Challenges](http://arxiv.org/abs/2402.03349v1)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2402.03349v1) |  |
| **[Differentiable modeling to unify machine learning and physical models and advance Geosciences](http://arxiv.org/abs/2301.04027v2)** | 2023-12-27 | [papers-cool](https://papers.cool/arxiv/2301.04027v2) |  |
| **[K2: A Foundation Language Model for Geoscience Knowledge Understanding and Utilization](http://arxiv.org/abs/2306.05064v2)** | 2023-09-13 | [papers-cool](https://papers.cool/arxiv/2306.05064v2) |  |
| **[AI Security for Geoscience and Remote Sensing: Challenges and Future Trends](http://arxiv.org/abs/2212.09360v2)** | 2023-06-22 | [papers-cool](https://papers.cool/arxiv/2212.09360v2) |  |
| **[Coarse-to-fine Task-driven Inpainting for Geoscience Images](http://arxiv.org/abs/2211.11059v3)** | 2022-12-06 | [papers-cool](https://papers.cool/arxiv/2211.11059v3) |  |
| **[Investigating the fidelity of explainable artificial intelligence methods for applications of convolutional neural networks in geoscience](http://arxiv.org/abs/2202.03407v2)** | 2022-09-05 | [papers-cool](https://papers.cool/arxiv/2202.03407v2) |  |
| **[Data Centred Intelligent Geosciences: Research Agenda and Opportunities, Position Paper](http://arxiv.org/abs/2209.02384v1)** | 2022-08-20 | [papers-cool](https://papers.cool/arxiv/2209.02384v1) |  |
| **[Beyond Visuals : Examining the Experiences of Geoscience Professionals With Vision Disabilities in Accessing Data Visualizations](http://arxiv.org/abs/2207.13220v1)** | 2022-07-27 | [papers-cool](https://papers.cool/arxiv/2207.13220v1) |  |

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Spatial Distribution-Shift Aware Knowledge-Guided Machine Learning](http://arxiv.org/abs/2502.14840v2)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2502.14840v2) |  |
| **[Fast Adversarial Training with Weak-to-Strong Spatial-Temporal Consistency in the Frequency Domain on Videos](http://arxiv.org/abs/2504.14921v2)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.14921v2) | <details><summary>After...</summary><p>After the submission of the paper, we realized that the study still has room for expansion. In order to make the research findings more profound and comprehensive, we have decided to withdraw the paper so that we can conduct further research and expansion</p></details> |
| **[Vulnerable Connectivity Caused by Local Communities in Spatial Networks](http://arxiv.org/abs/2412.14513v4)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2412.14513v4) | <details><summary>Modif...</summary><p>Modified from the first version</p></details> |
| **[Vision language models are unreliable at trivial spatial cognition](http://arxiv.org/abs/2504.16061v1)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.16061v1) |  |
| **[Recent Advances and Future Directions in Extended Reality (XR): Exploring AI-Powered Spatial Intelligence](http://arxiv.org/abs/2504.15970v1)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.15970v1) | 7 pages,4 figures |
| **[Visual Place Cell Encoding: A Computational Model for Spatial Representation and Cognitive Mapping](http://arxiv.org/abs/2504.15953v1)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.15953v1) |  |
| **[Joint leave-group-out cross-validation in Bayesian spatial models](http://arxiv.org/abs/2504.15586v1)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.15586v1) |  |
| **[Location-Oriented Sound Event Localization and Detection with Spatial Mapping and Regression Localization](http://arxiv.org/abs/2504.08365v2)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.08365v2) |  |
| **[CAPTURe: Evaluating Spatial Reasoning in Vision Language Models via Occluded Object Counting](http://arxiv.org/abs/2504.15485v1)** | 2025-04-21 | [papers-cool](https://papers.cool/arxiv/2504.15485v1) | <details><summary>Code ...</summary><p>Code and data: https://github.com/atinpothiraj/CAPTURe</p></details> |
| **[Explorable INR: An Implicit Neural Representation for Ensemble Simulation Enabling Efficient Spatial and Parameter Exploration](http://arxiv.org/abs/2504.00904v2)** | 2025-04-21 | [papers-cool](https://papers.cool/arxiv/2504.00904v2) | <details><summary>Accep...</summary><p>Accepted by IEEE Transactions on Visualization and Computer Graphics (TVCG)</p></details> |
| **[STI-Bench: Are MLLMs Ready for Precise Spatial-Temporal World Understanding?](http://arxiv.org/abs/2503.23765v3)** | 2025-04-21 | [papers-cool](https://papers.cool/arxiv/2503.23765v3) |  |
| **[A Call for New Recipes to Enhance Spatial Reasoning in MLLMs](http://arxiv.org/abs/2504.15037v1)** | 2025-04-21 | [papers-cool](https://papers.cool/arxiv/2504.15037v1) |  |
| **[OmniAudio: Generating Spatial Audio from 360-Degree Video](http://arxiv.org/abs/2504.14906v1)** | 2025-04-21 | [papers-cool](https://papers.cool/arxiv/2504.14906v1) | Work in Progress |
| **[Segmentation with Noisy Labels via Spatially Correlated Distributions](http://arxiv.org/abs/2504.14795v1)** | 2025-04-21 | [papers-cool](https://papers.cool/arxiv/2504.14795v1) |  |
| **[PH2ST:ST-Prompt Guided Histological Hypergraph Learning for Spatial Gene Expression Prediction](http://arxiv.org/abs/2503.16816v2)** | 2025-04-20 | [papers-cool](https://papers.cool/arxiv/2503.16816v2) |  |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Building Real-time Awareness of Out-of-distribution in Trajectory Prediction for Autonomous Vehicles](http://arxiv.org/abs/2409.17277v2)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2409.17277v2) |  |
| **[Regression-based proximal causal inference for right-censored time-to-event data](http://arxiv.org/abs/2409.08924v4)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2409.08924v4) | 14 pages, 2 figures |
| **[Distributed Unknown Input Observers for Discrete-Time Linear Time-Invariant Systems](http://arxiv.org/abs/2504.16815v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16815v1) |  |
| **[EvTTC: An Event Camera Dataset for Time-to-Collision Estimation](http://arxiv.org/abs/2412.05053v3)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2412.05053v3) | <details><summary>10 pa...</summary><p>10 pages, 7 figures, 5 tables</p></details> |
| **[A Time Series Dataset of NIR Spectra and RGB and NIR-HSI Images of the Barley Germination Process](http://arxiv.org/abs/2504.16658v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16658v1) |  |
| **[FREAK: Frequency-modulated High-fidelity and Real-time Audio-driven Talking Portrait Synthesis](http://arxiv.org/abs/2503.04067v2)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2503.04067v2) | <details><summary>Accep...</summary><p>Accepted by ICMR 2025</p></details> |
| **[ToF-Splatting: Dense SLAM using Sparse Time-of-Flight Depth and Multi-Frame Integration](http://arxiv.org/abs/2504.16545v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16545v1) |  |
| **[Con4m: Context-aware Consistency Learning Framework for Segmented Time Series Classification](http://arxiv.org/abs/2408.00041v3)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2408.00041v3) |  |
| **[Dynamic Time-aware Continual User Representation Learning](http://arxiv.org/abs/2504.16501v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16501v1) |  |
| **[Harden and Catch for Just-in-Time Assured LLM-Based Software Testing: Open Research Challenges](http://arxiv.org/abs/2504.16472v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16472v1) | <details><summary>To Ap...</summary><p>To Appear as keynote paper at FSE 2025</p></details> |
| **[EAGLE-3: Scaling up Inference Acceleration of Large Language Models via Training-Time Test](http://arxiv.org/abs/2503.01840v3)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2503.01840v3) |  |
| **[iTFKAN: Interpretable Time Series Forecasting with Kolmogorov-Arnold Network](http://arxiv.org/abs/2504.16432v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16432v1) |  |
| **[Hierarchical Distributed Architecture for the Least Allan Variance Atomic Timing](http://arxiv.org/abs/2504.16413v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16413v1) |  |
| **[A Robust Real-Time Lane Detection Method with Fog-Enhanced Feature Fusion for Foggy Conditions](http://arxiv.org/abs/2504.06121v3)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.06121v3) |  |
| **[Real-time Bayesian inference at extreme scale: A digital twin for tsunami early warning applied to the Cascadia subduction zone](http://arxiv.org/abs/2504.16344v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16344v1) |  |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Building Real-time Awareness of Out-of-distribution in Trajectory Prediction for Autonomous Vehicles](http://arxiv.org/abs/2409.17277v2)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2409.17277v2) |  |
| **[DYNUS: Uncertainty-aware Trajectory Planner in Dynamic Unknown Environments](http://arxiv.org/abs/2504.16734v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16734v1) | <details><summary>20 pa...</summary><p>20 pages, 30 figures, Under review at IEEE Transactions on Robotics</p></details> |
| **[TraCeS: Trajectory Based Credit Assignment From Sparse Safety Feedback](http://arxiv.org/abs/2504.12557v2)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.12557v2) |  |
| **[SILM: A Subjective Intent Based Low-Latency Framework for Multiple Traffic Participants Joint Trajectory Prediction](http://arxiv.org/abs/2504.16377v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16377v1) |  |
| **[Text-to-TrajVis: Enabling Trajectory Data Visualizations from Natural Language Questions](http://arxiv.org/abs/2504.16358v1)** | 2025-04-23 | [papers-cool](https://papers.cool/arxiv/2504.16358v1) |  |
| **[Dynamic Intent Queries for Motion Transformer-based Trajectory Prediction](http://arxiv.org/abs/2504.15766v1)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.15766v1) |  |
| **[Equi-Euler GraphNet: An Equivariant, Temporal-Dynamics Informed Graph Neural Network for Dual Force and Trajectory Prediction in Multi-Body Systems](http://arxiv.org/abs/2504.13768v2)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.13768v2) | <details><summary>permi...</summary><p>permission not yet received for arXiv</p></details> |
| **[SocialMOIF: Multi-Order Intention Fusion for Pedestrian Trajectory Prediction](http://arxiv.org/abs/2504.15616v1)** | 2025-04-22 | [papers-cool](https://papers.cool/arxiv/2504.15616v1) | 11 pages,6 figures |
| **[Offline Safe Reinforcement Learning Using Trajectory Classification](http://arxiv.org/abs/2412.15429v5)** | 2025-04-19 | [papers-cool](https://papers.cool/arxiv/2412.15429v5) | <details><summary>AAAI ...</summary><p>AAAI 2025. Updated results</p></details> |
| **[DiffOG: Differentiable Policy Trajectory Optimization with Generalizability](http://arxiv.org/abs/2504.13807v1)** | 2025-04-18 | [papers-cool](https://papers.cool/arxiv/2504.13807v1) |  |
| **[Learning Through Retrospection: Improving Trajectory Prediction for Automated Driving with Error Feedback](http://arxiv.org/abs/2504.13785v1)** | 2025-04-18 | [papers-cool](https://papers.cool/arxiv/2504.13785v1) |  |
| **[Lightweight LiDAR-Camera 3D Dynamic Object Detection and Multi-Class Trajectory Prediction](http://arxiv.org/abs/2504.13647v1)** | 2025-04-18 | [papers-cool](https://papers.cool/arxiv/2504.13647v1) |  |
| **[Stochastic Trajectory Optimization for Robotic Skill Acquisition From a Suboptimal Demonstration](http://arxiv.org/abs/2408.03131v4)** | 2025-04-18 | [papers-cool](https://papers.cool/arxiv/2408.03131v4) |  |
| **[Robust Estimation of Battery State of Health Using Reference Voltage Trajectory](http://arxiv.org/abs/2504.13324v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13324v1) |  |
| **[Uncertainty-Aware Trajectory Prediction via Rule-Regularized Heteroscedastic Deep Classification](http://arxiv.org/abs/2504.13111v1)** | 2025-04-17 | [papers-cool](https://papers.cool/arxiv/2504.13111v1) | <details><summary>17 Pa...</summary><p>17 Pages, 9 figures. Accepted to Robotics: Science and Systems(RSS), 2025</p></details> |
